Sources/PackageCollections/Model/TargetListResult.swift

```diff
@@ -33,8 +33,11 @@ extension PackageCollectionsModel.TargetListResult {
     public struct Package: Hashable, Encodable {
         public typealias Version = PackageCollectionsModel.TargetListResult.PackageVersion
 
-        /// Package's repository address
-        public let repository: RepositorySpecifier
+        /// Package's identity
+        public let identity: PackageIdentity
+
+        /// Package's location
+        public let location: String
 
```

Comment:
Ditto ^. Cannot remove `repository` until rdar://83113368 and rdar://83114538 are done.

Reply:
@yim-lee does this help: https://github.com/apple/swift-package-manager/pull/3743/commits/dcb79b3a5608886d372cdea2b05b8b232258f569